<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AMLinePropertyEditor</class>
 <widget class="QFrame" name="AMLinePropertyEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>159</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AMLinePropertyEditor {
	background-color: rgba(0, 0, 0, 179);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelColor">
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 230, 230);</string>
       </property>
       <property name="text">
        <string>Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AMColorPickerButton" name="colorButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelWidth">
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 230, 230);</string>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="widthSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="lineStyleBox">
     <item>
      <property name="text">
       <string>⎯⎯⎯⎯⎯⎯⎯</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>—  —  —  —  —</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>· · · · · · · · · · ·</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>— · — · — · —</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> — · · — · · —</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="labelFill">
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 230, 230);</string>
       </property>
       <property name="text">
        <string>Area Fill</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="fillCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelFillColor">
       <property name="styleSheet">
        <string notr="true">color: rgb(230, 230, 230);</string>
       </property>
       <property name="text">
        <string>Fill Color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AMColorPickerButton" name="fillColorButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AMColorPickerButton</class>
   <extends>QToolButton</extends>
   <header>ui/AMColorPickerButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
